.github/workflows/python-package.yml

```diff
@@ -54,3 +54,10 @@ jobs:
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
         run: twine upload */*
+      - name: Upload release artifact
+        uses: softprops/action-gh-release@v0.1.14
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            dist/*.tar.gz
+            dist/*.whl
```

Comment:
In this job (after the download artifacts action) I don't believe these files are under a dist directory. Also the entire `publish-package` job is guarded with a tag condition, so the one on this step is redundant.